proposals/0201-high-level-interface-overlay-controllers.md

```diff
@@ -172,7 +172,7 @@ To realize above behavior, the app developer must create a new view and place it
 @end
 ```
 
-If the `opaque` flag is set to NO the view will be called transparent. It should allow views or view parts from the underlying view controller to stay visible while the overlay controller is presenting. The opaque mode is view type related and allows . With this flag:
+If the `opaque` flag is set to NO the view will be called transparent. It should allow views or view parts from the underlying view controller to stay visible while the overlay controller is presenting. The opaque mode is view type related and allows overriding content per view. As an example a transparend overlay, which contains a graphic will only override the graphic view of the underlying view. With this flag:
```

Comment:
Update to:
If the `opaque` flag is set to NO the view will be called transparent. It should allow views or view parts from the underlying view controller to stay visible while the overlay controller is presenting. The opaque mode is view type related and allows overriding content per view. **As an example,** a **transparent** overlay contains a graphic that will only override the graphic view of the underlying view. With this flag: